Microsoft.Toolkit.Wpf.UI.XamlHost/IDesktopWindowXamlSourceNative2.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.InteropServices;
+using WUX = Windows.UI.Xaml;
+
+namespace Microsoft.Toolkit.Win32.UI.XamlHost
+{
+    /// <summary>
+    /// Enables access to native methods on DesktopWindowXamlSourceNative version 2 for RS5
+    /// </summary>
+    /// <remarks>
+    /// Includes the method used to set the window handle of the <see cref="WUX.Hosting.DesktopWindowXamlSource" /> instance.
+    /// </remarks>
+    [ComImport]
+    [InterfaceType(ComInterfaceType.InterfaceIsIUnknown)]
+    [Guid("e3dcd8c7-3057-4692-99c3-7b7720afda31")]
+    internal partial interface IDesktopWindowXamlSourceNative2
```

Comment:
Duplicated file at Microsoft.Toolkit.Forms.UI.XamlHost/IDesktopWindowXamlSourceNative2.cs

Reply:
The type of message is different for WPF and for WinForms, each framework requires a different copy.

---
In reply to: [261020079](https://github.com/windows-toolkit/Microsoft.Toolkit.Win32/pull/85#discussion_r261020079) [](ancestors = 261020079)